### ISSUE TITLE
set preview and edit mode at the same time

### ISSUE THREAD

**Issue by imeed166** [bwydoogh/obsidian-force-view-mode-of-note#24]
## New configuration 
```yaml
obsidianUIMode: preview
obsidianEditingMode: source
```
or
```yaml
obsidianUIMode: preview
obsidianEditingMode: live
```
## Intended behavior
The notes open in preview mode, but when switching to edit mode, the note go directly to the mode set in the other option (either live or source mode)

Could this be implemented, or would it complicate the plugin too much ?

**Comment by imeed166** [comment]
This could be useful when using live preview for most notes (default obsidian configuration : `source live`)

But there is a note that contains multiple dataview codeblocks. You want to view this note in preview mode, but when modifying the note, you want to switch to source editing mode for easier editing.

**Comment by bwydoogh** [comment]
I'm not sure I fully understand what you want to say. When using below combination the note opens in preview; when I switch to edit mode it goes into "source" edit mode (while my Obsidian editing mode is set to "Live Preview").

```
obsidianUIMode: preview
obsidianEditingMode: source
```

You are on the latest version of the plug-in?

**Comment by imeed166** [comment]
Yes, I'm on the latest version. My bad, I didn't bother to try it. It already works exactly like I wanted, thank you. (might want to add this to the README though)

**Comment by imeed166** [comment]
Wait, I think I found a bug.

obsidian configuration : 
- default new pane mode : editing view
- default editing mode : live preview

I have `Note1` with this 
```yaml
obsidianUIMode: preview
obsidianEditingMode: source
```
and `Note2` without yaml

---
1. go to `Note1` and switch to edit mode
2. go to `Note2`, the editing mode is still `source`, it doesn't switch to the default `live` even though the note doesn't contain yaml

Is this how it's supposed to behave ? (If I need to create another issue for this, tell me)

**Comment by bwydoogh** [comment]
> Is this how it's supposed to behave

No; it is a bug. Will have a look at it. Thanks!

**Comment by bwydoogh** [comment]
Fixed in `1.1.1`
